### PR TITLE
Documentation update on XTS

### PR DESCRIPTION
**Pull request by DVNorbertAcatrinei**
[AB#2018](https://dev.azure.com/data-vault/dbtvault/_workitems/edit/2018/)

- Refactoring table names in the sample SQL in macros.md
- Updated (per model - yaml strings) details in metadata.md
- Updated the XTS tutorial

### CODE REVIEW

File: docs/macros.md [hunk 1]
```diff
@@ -959,15 +959,15 @@ ___
 
 ### xts
 
-([view source](https://github.com/Datavault-UK/dbtvault/blob/v0.7.4/macros/tables/xts.sql))
```
Comment (DVAlexHiggs) [inline docs/macros.md:962]: Please change this. 

We should be using the ([view source]) line. Just change it to 0.7.9. It should just be versioned in advance and expectation of the next release.

File: docs/macros.md [hunk 2]
```diff
@@ -991,24 +994,28 @@ Generates SQL to build an Extended Tracking Satellite table using the provided p
     === "Single-Source"
 
         ```sql
-        WITH 
-        satellite_SATELLITE_1_from_PRIMED_STAGE AS (
+        WITH satellite_SATELLITE_1_from_STG_CUSTOMER AS (
```
Comment (DVAlexHiggs) [inline docs/macros.md:997]: These should be easily readable names. If this is copy and pasted from actual SQL that's great, but some minor modifications should be made to make the names of things more realistic.

File: docs/tutorial/tut_xts.md [hunk 3]
```diff
@@ -29,29 +29,36 @@ which is the code for `stg_customer`)
...
+       xts:
+        materialized: incremental
+        tags:
+          - xts
+        xts_customer:
+          vars:
+            ...
+    ```
+
+[comment]: <> (!!! tip "New in dbtvault v0.7.0")
```
Comment (DVAlexHiggs) [inline docs/tutorial/tut_xts.md:57]: Remove these comments please.

Comment (DVNorbertAcatrinei) [reply docs/tutorial/tut_xts.md:57]: Removed the comments. 

The reason I commented them out in the first place, was that I was not sure whether having a Tip starting with a "New in dbtvault **0.7.0**" title is still appropriate. Let me know if you want to change/remove the Tip's title.

File: docs/macros.md (hunk 2 above)
Comment (DVNorbertAcatrinei) [reply docs/macros.md:997]: Renamed the satellite CTEs to "satellite_a", "satellite_b" etc. 

Indeed. These CTEs need a better name "formula".

File: docs/macros.md (hunk 1 above)
Comment (DVNorbertAcatrinei) [reply docs/macros.md:962]: Uncommented and changed the version to ".../v0.7.9/..."

File: docs/macros.md (hunk 2 above)
Comment (DVAlexHiggs) [reply docs/macros.md]: Let's remove the tip entirely.

**Pull request merged**